spec/index.md

```diff
@@ -3723,26 +3801,28 @@ State after
 ```html
 
 S with
-    canisters[CanisterId] = EmptyCanister
```

Comment:
I changed `CanisterId` to `Canister_id` to have consistent naming with the other canister creation algorith.